CHANGES.rst

```diff
@@ -3,9 +3,116 @@
 0.14 (unreleased)
 -----------------
 
-- Planned for September 15, 2019
+Summary
++++++++
+
+- Released on September 30th, 2019
+- 8 contributors
+- 95 pull requests (not all listed below)
+
+**What's new**
+
+Gammapy v0.14 features a new high level analysis interface. Starting from
+a YAML configuration file, it supports the standard use-cases of joined
+or stacked 3D as well as 1D reflected region analyses. It also supports
+computation of flux points for all cases. The usage of this new ``Analysis``
+class is demonstrated in the ``hess.ipynb`` tutorial.
```

Comment:
Maybe put a Sphinx link to the tutorial? I know we usually don't do this in the changelog, but for now this will be the most valuable part of the what's new message, and if this breaks in the future we will get a Sphinx warning or error and can change it to a non-link.

Reply:
Done